include/dbclean.php

```diff
@@ -18,13 +18,11 @@ function dbclean_run(&$argv, &$argc) {
 	}
 
 	if ($stage == 0) {
-		proc_run(PRIORITY_LOW, 'include/dbclean.php', 1);
-		proc_run(PRIORITY_LOW, 'include/dbclean.php', 2);
-		proc_run(PRIORITY_LOW, 'include/dbclean.php', 3);
-		proc_run(PRIORITY_LOW, 'include/dbclean.php', 4);
-		proc_run(PRIORITY_LOW, 'include/dbclean.php', 5);
-		proc_run(PRIORITY_LOW, 'include/dbclean.php', 6);
-		proc_run(PRIORITY_LOW, 'include/dbclean.php', 7);
+		for ($i = 1; $i <= 7; $i++) {
+			if (!Config::get('system', 'finished-dbclean-'.$i)) {
```

Comment:
Yes, this is an intended behaviour. Currently the system is full of orphaned data because we only delete parts. There had been places where only items were deleted - but not the item related data. With the last commit I replaced all these delete commands with the new one that automatically deletes all depending data.

This means that - theoretically - there will be no orphaned data left, once it was deleted. The only exception is the first case.